### ISSUE TITLE
Android crash - Failed to connect to api.instabug.com/:::443

### ISSUE THREAD

**Issue by subhash279** [Instabug/Instabug-Flutter#142]
Hi, 
This is causing a fatal crash on android. It's happening on live versions. Do you happen to know this issue.

```
Caused by java.net.ConnectException
Failed to connect to api.instabug.com/:::443

com.instabug.library.network.NetworkManager.connectWithNormalType (NetworkManager.java:8)
com.instabug.library.network.NetworkManager.access$100 (NetworkManager.java:1)
com.instabug.library.network.NetworkManager$a.subscribe (NetworkManager.java:20)
io.reactivex.internal.operators.observable.ObservableCreate.subscribeActual (ObservableCreate.java:40)
io.reactivex.Observable.subscribe (Observable.java:12051)
io.reactivex.internal.operators.observable.ObservableRetryPredicate$RepeatObserver.subscribeNext (ObservableRetryPredicate.java:111)
io.reactivex.internal.operators.observable.ObservableRetryPredicate.subscribeActual (ObservableRetryPredicate.java:41)
io.reactivex.Observable.subscribe (Observable.java:12051)
io.reactivex.internal.operators.observable.ObservableIgnoreElementsCompletable.subscribeActual (ObservableIgnoreElementsCompletable.java:31)
io.reactivex.Completable.subscribe (Completable.java:1919)
io.reactivex.internal.operators.completable.CompletablePeek.subscribeActual (CompletablePeek.java:51)
io.reactivex.Completable.subscribe (Completable.java:1919)
io.reactivex.internal.operators.completable.CompletableConcatArray$ConcatInnerObserver.next (CompletableConcatArray.java:89)
io.reactivex.internal.operators.completable.CompletableConcatArray.subscribeActual (CompletableConcatArray.java:33)
io.reactivex.Completable.subscribe (Completable.java:1919)
io.reactivex.internal.operators.completable.CompletableConcatArray$ConcatInnerObserver.next (CompletableConcatArray.java:89)
io.reactivex.internal.operators.completable.CompletableConcatArray.subscribeActual (CompletableConcatArray.java:33)
io.reactivex.Completable.subscribe (Completable.java:1919)
io.reactivex.internal.operators.completable.CompletableSubscribeOn$SubscribeOnObserver.run (CompletableSubscribeOn.java:64)
io.reactivex.Scheduler$DisposeTask.run (Scheduler.java:579)
io.reactivex.internal.schedulers.ScheduledRunnable.run (ScheduledRunnable.java:66)
io.reactivex.internal.schedulers.ScheduledRunnable.call (ScheduledRunnable.java:57)
```

**Comment by AliAbdelfattah** [comment]
Hi @subhash279, sorry you ran into this crash. I've created a snapshot that fixes it for you while you wait for the official release.

`https://github.com/Instabug/Instabug-Flutter/tree/custom/android-v9.1.6-fix-connection-exception`

**Comment by stwonary** [comment]
Seems to be related to https://github.com/Instabug/Instabug-Android/issues/320
And it's fixed in v9.1.7 https://github.com/Instabug/Instabug-Android/releases

**Comment by subhash27** [comment]
thanks for letting me know. Will this fix be pushed into this repo as well soon?

**Comment by AliAbdelfattah** [comment]
@subhash27 Yes. It should be fixed in the next official release of this repo.

**Comment by AliAbdelfattah** [comment]
Fixed in [v9.1.7](https://github.com/Instabug/Instabug-Flutter/releases/tag/v9.1.7).

**Comment by subhash279** [comment]
@AliAbdelfattah thank you so much. May I pls know when this update will be pushed to pub.dev package.
